docs/nvidia-guide.md

```diff
@@ -0,0 +1,78 @@
+# NVIDIA graphics cards
+_Driver API and source code available in [`liquidctl.driver.nvidia`](../liquidctl/driver/nvidia.py)._
+
+Support for these cards in only available on Linux.
+
+Additional requirements must also be met:
+
+- optional Python dependency `smbus` is available
+- `i2c-dev` kernel module has been loaded
```

Comment:
I should prepare and include a "load the `i2c-dev` module during boot" configuration file for inclusion in distro packages.